xbmc/cores/VideoPlayer/VideoRenderers/VideoShaders/YUV2RGBShaderGL.cpp

```diff
@@ -176,6 +176,8 @@ bool BaseYUV2RGBGLSLShader::OnEnabled()
     else if (m_hasDisplayMetadata && m_displayMetadata.has_luminance)
       param = log10(100) / log10(m_displayMetadata.max_luminance.num/m_displayMetadata.max_luminance.den);
```

Comment:
``` m_displayMetadata.max_luminance.num/m_displayMetadata.max_luminance.den ```
Take care this is an integer division and results in zero if den < num. Btw. I am not sure the algorithm is correct. I have far better results when using lum_max / lum_min